release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -296,6 +296,9 @@ Code Freeze will typically fall around Weeks 8 or 9 depending on the length or r
 ### Week 13
 
 - Release Retrospective participation
+  - Make sure to confirm a facilitator for the Release Retrospective with SIG PM
```

Comment:
@claurence, @lachie83, and I discussed having the Emeritus Adviser be the default facilitator for the retro, deferring the responsibility if they are going to be unavailable on the date.

ref: https://github.com/kubernetes/sig-release/issues/953